### ISSUE TITLE
UnicodeDecodeError in smbexec.py

### ISSUE THREAD

**Issue by exploide** [SecureAuthCorp/impacket#708]
When running `smbexec.py` (today's master version) and executing any command, an `UnicodeDecodeError` occurs. This is maybe due to some German umlauts, e.g. in `Domänen-Admins`. But perhaps it is just another issue with the `AttributeError: 'RemoteShell' object has no attribute 'do_net'`.

```
> smbexec.py -debug 'domain/user:pass@192.168.1.1'
Impacket v0.9.21-dev - Copyright 2019 SecureAuth Corporation

[+] StringBinding ncacn_np:192.168.1.1[\pipe\svcctl]
[+] Executing %COMSPEC% /Q /c echo cd  ^> \\127.0.0.1\C$\__output 2^>^&1 > %TEMP%\execute.bat & %COMSPEC% /Q /c %TEMP%\execute.bat & del %TEMP%\execute.bat
[!] Launching semi-interactive shell - Careful what you execute
C:\Windows\system32>net group
[+] Executing %COMSPEC% /Q /c echo net group ^> \\127.0.0.1\C$\__output 2^>^&1 > %TEMP%\execute.bat & %COMSPEC% /Q /c %TEMP%\execute.bat & del %TEMP%\execute.bat
Traceback (most recent call last):
  File "/usr/lib64/python3.7/cmd.py", line 214, in onecmd
    func = getattr(self, 'do_' + cmd)
AttributeError: 'RemoteShell' object has no attribute 'do_net'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jvieten/Repos/impacket/venv/bin/smbexec.py", line 152, in run
    self.shell.cmdloop()
  File "/usr/lib64/python3.7/cmd.py", line 138, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib64/python3.7/cmd.py", line 216, in onecmd
    return self.default(line)
  File "/home/jvieten/Repos/impacket/venv/bin/smbexec.py", line 241, in default
    self.send_data(line)
  File "/home/jvieten/Repos/impacket/venv/bin/smbexec.py", line 278, in send_data
    print(self.__outputBuffer.decode())
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x81 in position 17: invalid start byte
[-] 'utf-8' codec can't decode byte 0x81 in position 17: invalid start byte
```

**Comment by asolino** [comment]
Hey @exploide 
Can you change line 278 of `smbexec.py` for:
```
        print(self.__outputBuffer.decode(errors='replace'))
```
and retest?

let me know how it goes.

**Comment by exploide** [comment]
Hi @asolino 
Sure, I try to help with this.

Meanwhile I was able to reproduce this issue. The error occurs after I add a user that contains a non-ascii character like a German umlaut: `net user Dänny ABC-123456 /add`

Then, running `net group` fails and displays the exception shown above.

I tried your fix and it works partially. The good news is that it no longer crashes and returns the user list. Sadly, it does not contain the proper character. Instead of `Dänny`, it becomes `D�nny`.
